### PR TITLE
Nbo/no scenario yet

### PR DESCRIPTION
**Pull request by Nikeleos**

### CODE REVIEW

File: src/Dashboard/components/DashboardPlaceholder/DashboardPlaceholder.js [hunk 1]
```diff
@@ -25,32 +25,41 @@ const useStyles = makeStyles(theme => ({
...
+        </Typography>
+      </Grid>
       <Grid item>
         { icon !== undefined &&
           <div className={classes.iconContainer}>
             {icon}
           </div>
         }
         <Typography
-          component="h2"
```
Comment (COSMO-VMI) [inline src/Dashboard/components/DashboardPlaceholder/DashboardPlaceholder.js:39]: Why did you remove the h2 component?

File: src/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -23,35 +24,40 @@ const Dashboard = (props) => {
...
   } = props;
   const formattedUrl = url.replaceAll('<ScenarioName>', scenarioName).replaceAll('<ScenarioId>', scenarioId);
+  const { t } = useTranslation();
 
   // Handle optional status property
   const noRun = scenarioState === 'Created' || scenarioState === null;
   const runInProgress = scenarioState === 'Running';
   const hasError = scenarioState === 'Failed';
-  const isReady = scenarioState === undefined || scenarioState === 'Successful';
+  const isReady = (scenarioState === undefined || scenarioState === 'Successful') && noScenario === false;
```
Comment (COSMO-VMI) [inline src/Dashboard/Dashboard.js:37]: `noScenario` is not marked as `required`, so it can be `undefinied`, which is not handled here.
Either mark it required, or handle the `undefined` case.

**Review by COSMO-VMI — CHANGES_REQUESTED** [review]
Some fixes, but good job.

File: src/Dashboard/components/DashboardPlaceholder/DashboardPlaceholder.js (hunk 1 above)
Comment (Nikeleos) [reply src/Dashboard/components/DashboardPlaceholder/DashboardPlaceholder.js:39]: Beacause label case should be different from title case

File: src/Dashboard/Dashboard.js (hunk 2 above)
Comment (Nikeleos) [reply src/Dashboard/Dashboard.js:37]: Another solution, the initial chosen one : add a default value to noScenario props (false).

Comment (jreynard-code) [reply src/Dashboard/Dashboard.js:43]: could be replaced by `!noScenario` instead of  `noScenario === false `

**Review by jreynard-code — APPROVED** [review]
LGTM ;) just some syntactic sugar

File: src/Dashboard/Dashboard.js (hunk 2 above)
Comment (Nikeleos) [reply src/Dashboard/Dashboard.js:43]: Done
